internal/osbuild2/pwquality_conf_stage.go

```diff
@@ -0,0 +1,23 @@
+package osbuild2
+
+type PwqualityConfConfig struct {
+	Minlen   int `json:"minlen"`
+	Dcredit  int `json:"dcredit"`
+	Ucredit  int `json:"ucredit"`
+	Lcredit  int `json:"lcredit"`
+	Ocredit  int `json:"ocredit"`
+	Minclass int `json:"minclass"`
```

Comment:
All of these values are optional should be marked as 'omitempty` and most probably also defined as pointer values, because the "zero" value for the type would otherwise end up in the stage options and would be defined in the pwquality configuration.